gettsim/tests/test_tax_transfer.py

```diff
@@ -14,7 +14,7 @@
 from gettsim.typing import check_if_series_has_internal_type
 
 
-YEARS = [2019, 2020, 2021]
+YEARS = [2019]
```

Comment:
Why?

Reply:
The current test data only contains test cases for 2019. (No clue, why the code looped over 2019-2021 before)